packages/core/src/connectivity/scp-cf/index.ts

```diff
@@ -10,3 +10,10 @@ export * from './resilience-options';
 export * from './token-accessor';
 export * from './xsuaa-service';
 export * from './xsuaa-service-types';
+export * from './tenant';
+export {
+  RegisteredJWTClaimsUser,
+  Scope,
+  mapping as mappingUserFields,
+  userFromJwt
+} from './user';
```

Comment:
Please update the release notes if they are missing.

Reply:
yes